<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>861</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_pwd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_role">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>395</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnquery">
      <property name="text">
       <string>查询内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btninsert">
      <property name="text">
       <string>输入内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btndelete">
      <property name="text">
       <string>查看目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnupdate">
      <property name="text">
       <string>更新数据</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
